myceliald/src/http_client.rs

```diff
@@ -233,6 +236,11 @@ impl HttpClient {
         rx: &mut Receiver<HttpClientMessage>,
     ) -> Result<(), SectionError> {
         let mut interval = tokio::time::interval(Duration::from_secs(5));
+        // set first tick to 50ms from now, allowing main application to set sonnection details
+        // before first loop iteration
+        if let Some(delay) = tokio::time::Instant::now().checked_add(Duration::from_millis(50)) {
```

Comment:
Minor nit, but this should probably be in the spawn function instead, it better communicates that it's setup before entering the loop and this function only handles the loop itself.

Reply:
moved sleep to spawn, @wwkeyboard does it look better now?